ui/src/store.js

```diff
@@ -1,26 +1,17 @@
-import { createStore, applyMiddleware, compose } from 'redux';
-import { createLogger } from 'redux-logger';
+import { createStore, applyMiddleware } from 'redux';
 import thunk from 'redux-thunk';
 import rootReducer from './reducers';
+import { composeWithDevTools } from '@redux-devtools/extension';
 
-function initStore() {
-  // Logger MUST BE the last middleware
-  const middleware = [thunk, createLogger()];
```

Comment:
I've decided to remove `redux-logger`, which clutters the console (potentially hiding important warnings/errors), and to instead enable `redux-devtools` in production (i.e. not just in development).

So if you need to debug MXCuBE on a beamline, just install the Redux devtools extension: https://github.com/reduxjs/redux-devtools/tree/main/extension#installation. It's much easier to debug the Redux state with this extension than by trying to read the `redux-logger` logs.

Note that I've also enabled sourcemaps generation in `vite.config.js`, which should also make production debugging easier.

Reply:
I am not sure we can install browser extensions at the beamline. I mean there is always ways around, but I doubt it is a case of "just install". Maybe we will never even need this, so not sure it is worth reconsidering this now.

Reply:
Personally, I trust your judgement and I rarely used redux-logger anyway.

I don't know about the others, maybe they rely on redux-logger, maybe it is worth asking (on the mailing list?) if they mind the migration to redux-devtools or not.

Reply:
I usually relay on `redux-logger` for debugging, this removal will make my life a bit worse 😔. We cannot install extensions on the beamline as we don't have internet access

Reply:
I vote for making `redux-logger` an opt in via an environment variable. It _is_ crazy spammy, would be nice not to look at that spam all the time. But, an easy way to enable it would be certainly very useful for us at MAXIV.

If you add such a variable, please add documentation for it at some good place here: https://mxcubeweb.readthedocs.io/en/latest/dev/index.html